cub/cub/thread/thread_operators.cuh

```diff
@@ -642,82 +581,168 @@ struct SimdMul<__half>
 template <>
 struct SimdMul<__nv_bfloat16>
 {
-  using simd_type = __nv_bfloat162;
-
-  _CCCL_NODISCARD _CCCL_DEVICE _CCCL_FORCEINLINE __nv_bfloat162 operator()(__nv_bfloat162 a, __nv_bfloat162 b) const
+  _CCCL_NODISCARD _CCCL_DEVICE _CCCL_FORCEINLINE __nv_bfloat162
+  operator()([[maybe_unused]] __nv_bfloat162 a, [[maybe_unused]] __nv_bfloat162 b) const
   {
-#    if _CCCL_CUDACC_BELOW(12) && _CCCL_CUDA_COMPILER(NVHPC)
-    return __floats2bfloat162_rn(
-      __bfloat162float(a.x) * __bfloat162float(b.x), __bfloat162float(a.y) * __bfloat162float(b.y));
-#    else // ^^^ _CCCL_CUDACC_BELOW(12) && _CCCL_CUDA_COMPILER(NVHPC) ^^^ / vvv otherwise vvv
     NV_IF_TARGET(NV_PROVIDES_SM_80,
-                 (return __hmul2(a, b);),
-                 (return halves2bfloat162(__float2bfloat16(__bfloat162float(a.x) * __bfloat162float(b.x)),
-                                          __float2bfloat16(__bfloat162float(a.y) * __bfloat162float(b.y)));));
-#    endif // !_CCCL_CUDACC_BELOW(12) || !_CCCL_CUDA_COMPILER(NVHPC)
+                 (return __hmul2(a, b);), //
+                 (_CCCL_ASSERT(false, "Not supported on SM < 53"); return __nv_bfloat162{};));
   }
 };
 
 #  endif // defined(_CCCL_HAS_NVBF16)
 
 //----------------------------------------------------------------------------------------------------------------------
+// Predefined operators
+
+using cub::detail::is_one_of_v;
+
+template <typename ReductionOp, typename T>
+inline constexpr bool is_predefined_min_max_v =
```

Comment:
Remark: maybe the wording "known" or "cuda_std" is better than "predefined".
```suggestion
inline constexpr bool is_cuda_std_min_max_v =
```
Applies to more places.